Topic4Services/src/cartServices/ShoppingCartImp.java

```diff
@@ -0,0 +1,57 @@
+package cartServices;
+
+import items.Container;
+import items.Item;
+
+import java.util.List;
+
+import payments.Payment;
+
+public class ShoppingCartImp implements ShoppingCartService {
+
+	private Cart cart;
+	private Container container;

```

Comment:
This container stores all the products that the e-commerce site will be offering to the user using the Shopping Cart, isn't it? If that is the case I consider that it shouldn't be handled by the same service who is handling the cart, these are two different things, one is the cart and another the products that you can add to the cart.
Please let me know if I'm misunderstanding the idea